ultra/docs/setup.md

```diff
@@ -4,27 +4,26 @@ ULTRA can be run natively on your machine, or through a Docker container. See in
 
 ## Setup Natively
 
-The following steps can be ignored if you already have the SMARTS environment.
 ```sh
 # 1 - Navigate to the ULTRA directory.
 $ cd <path_to_SMARTS>/SMARTS/ultra
 
-# 2 - Create a virtual environment.
+# 2 - Install overhead dependencies (python3.7 & sumo)
+./install_deps.sh
+
+# 3 - verify sumo is >= 1.5.0
+# if you have issues see ./docs/SUMO_TROUBLESHOOTING.md
+sumo
```

Comment:
```suggestion
$ ./install_deps.sh

# 3 - verify sumo is >= 1.5.0
# if you have issues see ./docs/SUMO_TROUBLESHOOTING.md
$ sumo
```
The other commands in this setup have a '$' preceding them. It definitely doesn't have to be there, but maybe we should just keep a consistent notation (either with or without the '$' for all commands).

Reply:
No that is right, we should keep them consistent, nice catch!!